### DISCUSSION TITLE
FreshRSS and NetNewsWire login issue

### DISCUSSION THREAD

**Discussion by ghost** [FreshRSS/FreshRSS#4313] [Q&A] [answered]
Hello to everyone,

I've installed on my server FreshRSS and everthing it's ok.
I want to connect my feed to the NetNewsWire on my android phone since it supports FreshRSS.

I've turned on the access via API in the settings, but when I type my login credentials and the full path of the Greader.php on my website, NetNewsWire tells me that i'm unathorized to access.

The whole folder ./p permission is set on 744, so i wonder where is the problem...

Can you help me to find out where I am doing wrong? Thank you in advance!

I apologize if there isn't the right place, but i don't know who and where to ask for my problem 😅

**Comment by Alkarex** [comment, marked as answer]
Hello,
1) Check the auto tests by browsing to https://freshrss.example.net/api/
2) Ensure that you have set an API password for your user
3) Test with another client or manually using cURL https://freshrss.github.io/FreshRSS/en/developers/06_GoogleReader_API.html#google-reader-compatible-api
4) Check the logs in `./FreshRSS/data/users/_/log_api.txt`

**Comment by ghost** [comment]
Hello Alkarex!

Thank you for the reply!
My bad, I didn't notice that under the profile section, there is a specific password for the API access. I used the "normal one", so, that was the problem.

Indeed, setup the API password and everthing goes ok! 
Thank you very much and have a nice day!
